src/platform/forms/components/review/SubmitButtons.jsx

```diff
@@ -0,0 +1,45 @@
+// libs
+import React from 'react';
+
+// platform - forms-system components
+import Column from 'platform/forms/components/common/grid/Column';
+import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
+import Row from 'platform/forms/components/common/grid/Row';
+
+// platform - forms containers
+import PreSubmitSection from 'platform/forms/containers/review/PreSubmitSection';
+
+function SubmitButtons(props) {
+  const { goBack, formConfig, onSubmit } = props;
+
+  return (
+    <>
+      <Row>
+        <Column />
+      </Row>
+      <PreSubmitSection formConfig={formConfig} />
+      <Row classNames="form-progress-buttons">
+        <Column classNames="small-6 medium-5">
+          <ProgressButton
+            onButtonClick={goBack}
+            buttonText="Back"
+            buttonClass="usa-button-secondary"
+            beforeText="«"
+          />
+        </Column>
+        <Column classNames="small-6 medium-5">
+          <ProgressButton
+            onButtonClick={onSubmit}
+            buttonText="Submit application"
+            buttonClass="usa-button-primary"
+          />
+        </Column>
+        <Column classNames="small-1 medium-1 end">
+          <div className="hidden">&nbsp;</div>
+        </Column>

```

Comment:
I see this is in the old version as well, so I won't suggest changing anything for now, but in a future iteration, we should figure out why this is here if it's actually hidden.

Reply:
I completely agree and I think this is somewhat related to the empty column above. I left it that way in this iteration as it felt out the scope of the refactor.